app/views/admin/articles/_errors.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @article.errors.any? %>
```

Comment:
This won't work if the errors come from @comment for example.   This can go in a shared folder `/views/shared/_errors.html.erb`  you will then need to use a local variable for the partial, see https://guides.rubyonrails.org/layouts_and_rendering.html#passing-local-variables for how this is done